snap/validate_test.go

```diff
@@ -592,3 +592,79 @@ func (s *ValidateSuite) TestValidateSocketName(c *C) {
 		c.Assert(err, ErrorMatches, `invalid socket name: ".*"`)
 	}
 }
+
+func (s *ValidateSuite) TestValidateStartupOrder(c *C) {
+
```

Comment:
This functions looks quite empty :)

Reply:
Thanks for spotting this. Fixed in the next commit.